features/taxation/applying_taxes_based_on_order_item_units_calculation_strategy/applying_correct_taxes_for_item_units_with_different_tax_rates.feature

```diff
@@ -0,0 +1,30 @@
+@applying_taxes
+Feature: Applying correct taxes for item units with different tax rates
+    In order to pay proper amount when buying goods from different tax categories
+    As a Visitor
+    I want to have correct taxes applied to my order
```

Comment:
I'm uncertain if a visitor is a good "actor" here. As a customer, I don't care whether the store calculates tax correctly, as I'm not paying them directly. 
```suggestion
    In order to pay proper amount of taxes
    As a Store Owner
    I want to have correct taxes applied for orders with items from different tax categories
```
I'm wondering about something like that.